### ISSUE TITLE
[Help wanted] `@farmfe/js-plugin-dts` how to map alias

### ISSUE THREAD

**Issue by niuiic** [farm-fe/farm#702]
### Steps to reproduce

```
git clone https://github.com/niuiic/flow
pnpm i
cd packages/fx-flow
pnpm build
```

Twice build for esm and cjs. The last one will fail because of using unsupported plugin `rollup-plugin-dts`.

Check `dist/esm/pipe/strict/consume.d.ts`, path alias `@` was not mapped to relative path.

### What is expected?

`import type { IterableReturnValue, UniversalIterable } from '@/types';`

To

`import type { IterableReturnValue, UniversalIterable } from '../../types';`

### What is actually happening?

No path alias was mapped in declare files.

### System Info

```Shell
System:
    OS: Linux 6.1 Gentoo Linux
    CPU: (16) x64 AMD Ryzen 7 5800H with Radeon Graphics
    Memory: 13.08 GB / 15.59 GB
    Container: Yes
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.6.1 - /run/user/1000/fnm_multishells/72785_1698977614366/bin/node
    npm: 9.8.1 - /run/user/1000/fnm_multishells/72785_1698977614366/bin/npm
    pnpm: 8.10.2 - /run/user/1000/fnm_multishells/72785_1698977614366/bin/pnpm
  npmPackages:
    @farmfe/core: ^0.13.19 => 0.13.19
```
```


### Any additional comments?

_No response_

**Comment by ErKeLost** [comment]
The dts plugin has not yet implemented the handling of alias. I will add this implementation on weekends

**Comment by niuiic** [comment]
Thx

**Comment by ErKeLost** [comment]
> Thx

fixed in @farmfe/js-plugin-dts@0.2.1
